setup.py

```diff
@@ -47,6 +47,23 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
+try:
+    import django
+    os.environ.setdefault(
+        "DJANGO_SETTINGS_MODULE",
+        "analytical.tests.settings"
+    )
+    django.setup()
+except ImportError:
+    print(
+        "Could not import django."
+        "This is fine, unless you intend to run unit tests"
+    )
+except AttributeError:
+    # django.setup() does not exist on older djangos, so don't worry about
+    # that.
+    pass
```

Comment:
You're right. This change was before dropping django < 1.7.

There is no `UnsupportedVersionError` in python, and I'd rather just get rid of the exception and raise that - in any case, supported versions should be well documented.